rts/lra/lra-test/pom.xml

```diff
@@ -35,14 +35,14 @@
              -Xrunjdwp:transport=dt_socket,address=${coordinator.debug.port},server=y,suspend=y
```

Comment:
do we need here this comment when the swarm.debug.jdwp attribute covers this?

Reply:
I just meant if information is not doubled and if you consider so the comment could be removed competely.

Reply:
okay